CODE_OF_CONDUCT.md

```diff
@@ -1,44 +0,0 @@
-# Contributor Code of Conduct
```

Comment:
Why have you deleted the Code of Conduct file? Please revert this.

Reply:
https://github.com/Cloud-CV/EvalAI-ngx/pull/111#discussion_r267420754
What were you asking to remove in that comment?